website/mkdocs-dyne-theme/.github/README.md

```diff
@@ -0,0 +1,101 @@
+# Dyne.org theme for software webpages made in mkdocs
```

Comment:
Do we really need to checkin this file I wonder...

Reply:
no need, can either be a git submodule or leave a script there which clones the repo locally. Another option which I personally adopted while developing the theme is to symlink the git clone on my own filesystem.